app/Http/Controllers/UsersController.php

```diff
@@ -130,10 +130,8 @@ public function postCreate(Request $request)
             }
             return redirect::route('users')->with('success', trans('admin/users/message.success.create'));
         } else {
-            redirect()->back()->withInput()->withInput()->withErrors($user->getErrors())->withErrors($settings->getErrors());
```

Comment:
Oh, wait, it might be - hard to tell without the full context :-/ I'll check it out